securedrop_client/gui/main.py

```diff
@@ -197,3 +198,10 @@ def clear_error_status(self):
         Clear any message currently in the error status bar.
         """
         self.top_pane.clear_error_status()
+
+    def clear_clipboard(self):
+        """
+        Purge any clipboard contents.
+        """
+        cb = QApplication.clipboard()
+        cb.clear()
```

Comment:
In reading the docs on `QApplication.clipboard` calling `clear` with the default options will indeed clear the global system clipboard contents, so this approach is sound. If you want to add tests for this I'll review formally tomorrow AM for inclusion in 0.2.0.